subworkflows/local/gtdbtk.nf

```diff
@@ -76,14 +78,17 @@ workflow GTDBTK {
 
     GTDBTK_CLASSIFYWF (
         ch_filtered_bins.passed.groupTuple(),
-        ch_db_for_gtdbtk
+        ch_db_for_gtdbtk,
+        gtdb_mash
     )
 
     GTDBTK_SUMMARY (
         ch_filtered_bins.discarded.map{it[1]}.collect().ifEmpty([]),
         GTDBTK_CLASSIFYWF.out.summary.collect().ifEmpty([]),
-        GTDBTK_CLASSIFYWF.out.filtered.collect().ifEmpty([]),
-        GTDBTK_CLASSIFYWF.out.failed.collect().ifEmpty([])
+        [],
+        // GTDBTK_CLASSIFYWF.out.filtered.collect().ifEmpty([]),
+        []
+        // GTDBTK_CLASSIFYWF.out.failed.collect().ifEmpty([])
```

Comment:
Why are those two commented out? Does that mean there is now information missing in the summary?

Reply:
@maxibor is still in the process of sorting a bunch of related problems out, so this release is only includes some of the fix.es However these will be cleaned up in the future (being to left to ensure testing/comparison later).